test/unit/import_playbooks_test.rb

```diff
@@ -0,0 +1,50 @@
+require 'test_plugin_helper'
+
+class ImportPlaybooksTest < ActiveSupport::TestCase
+  let(:ansible_proxy) { FactoryBot.create(:smart_proxy, :with_ansible) }
+  let(:proxy_api) { ::ProxyAPI::Ansible.new(url: ansible_proxy.url) }
+  let(:first_playbook) { ['xprazak2.forklift_collection.foreman_provisioning.yml'] }
+  let(:second_playbook) { ['xprazak2.forklift_collection.collect_debug_draft.yml'] }
+  subject { ::ForemanAnsible::PlaybooksImporter.new(ansible_proxy) }
+  let(:sample_playbooks) { JSON.parse(File.read(ansible_fixture_file('sample_playbooks.json'))) }
+  let(:playbook1_output) { sample_playbooks['1'] }
+  let(:playbook2_output) { sample_playbooks['2'] }
+
+  setup do
+    subject.stubs(:proxy_api).returns(proxy_api)
+  end
+
+  describe 'check the job templates that were created due to the import' do
+    test 'should just create job templates from playbooks' do
+      subject.expects(:playbooks).with(second_playbook).returns([playbook2_output])
+      actual = subject.import_playbooks(second_playbook)
+      assert_not_nil actual_created = actual[:created]
+      assert_equal 1, actual_created.count
+      job_template = JobTemplate.where(id: actual_created.keys.first).first
+      assert_equal job_template.name, actual_created.values.first
+    end
+  end
+
+  describe 'check the updated job templated due to import playbooks' do
+    test ' should not update an identical job template' do
+      FactoryBot.create(:job_template, id: 111, name: 'xprazak2.forklift_collection.foreman_provisioning.yml',
+                                       template: "      <%#\n        name: xprazak2.forklift_collection.foreman_provisioning.yml\n        snippet: false\n        job_category: Ansible Playbook - Imported\n        provider_type: Ansible\n        kind: job_template\n        model: JobTemplate\n      %>\n- hosts: all\n  become: true\n  vars:\n    libvirt_tftp: true\n  roles:\n    - foreman\n    - libvirt\n    - foreman_provisioning\n")
```

Comment:
can't we get this data (`name` and `template`) from the fixtures?

Reply:
I added a new file for the template because in the near future we will need to add more exampled output for checking the inputs/variables